api/src/api/v2.js

```diff
@@ -203,6 +205,7 @@ router.ws('/connect', async (ws, req) => {
 
     ws.on('message', async data => {
         try {
+            // @ts-ignore
             const msg = JSON.parse(data);
```

Comment:
JSON.parse takes in `string` but `data` is typed `Buffer | ArrayBuffer | Buffer[]`. Internally it invokes `toString` and in production it does work.